viz/tailwind.config.ts

```diff
@@ -0,0 +1,20 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
```

Comment:
This does not seem to be targeting the right path for the app router hierarchy.

Reply:
indeed